src/api.js

```diff
@@ -738,171 +738,139 @@ Database = (function() {
-
-sqlite3_column_type = Module['cwrap']('sqlite3_column_type', 'number', ['number', 'number']);
-
-sqlite3_column_name = Module['cwrap']('sqlite3_column_name', 'string', ['number', 'number']);
-
-sqlite3_reset = Module['cwrap']('sqlite3_reset', 'number', ['number']);
-
-sqlite3_clear_bindings = Module['cwrap']('sqlite3_clear_bindings', 'number', ['number']);
-
-sqlite3_finalize = Module['cwrap']('sqlite3_finalize', 'number', ['number']);
-
-sqlite3_create_function_v2 = Module['cwrap']('sqlite3_create_function_v2', 'number', ['number', 'string', 'number', 'number', 'number', 'number', 'number', 'number', 'number']);
-
-sqlite3_value_type = Module['cwrap']('sqlite3_value_type', 'number', ['number']);
-
-sqlite3_value_bytes = Module['cwrap']('sqlite3_value_bytes', 'number', ['number']);
-
-sqlite3_value_text = Module['cwrap']('sqlite3_value_text', 'string', ['number']);
-
-sqlite3_value_int = Module['cwrap']('sqlite3_value_int', 'number', ['number']);
-
-sqlite3_value_blob = Module['cwrap']('sqlite3_value_blob', 'number', ['number']);
-
-sqlite3_value_double = Module['cwrap']('sqlite3_value_double', 'number', ['number']);
-
-sqlite3_result_double = Module['cwrap']('sqlite3_result_double', '', ['number', 'number']);
-
-sqlite3_result_null = Module['cwrap']('sqlite3_result_null', '', ['number']);
-
-sqlite3_result_text = Module['cwrap']('sqlite3_result_text', '', ['number', 'string', 'number', 'number']);
-
-sqlite3_result_blob = Module['cwrap']('sqlite3_result_blob', '', ['number', 'number', 'number', 'number']);
-
-sqlite3_result_int = Module['cwrap']('sqlite3_result_int', '', ['number', 'number']);
-
-sqlite3_result_int64 = Module['cwrap']('sqlite3_result_int64', '', ['number', 'number']);
-
-sqlite3_result_error = Module['cwrap']('sqlite3_result_error', '', ['number', 'string', 'number']);
-
-RegisterExtensionFunctions = Module['cwrap']('RegisterExtensionFunctions', 'number', ['number']);
-
+/*jslint browser*/
+/*global NULL SQLite*/
+var RegisterExtensionFunctions;
+var cwrap;
+var sqlite3_bind_blob;
+var sqlite3_bind_double;
+var sqlite3_bind_int;
+var sqlite3_bind_parameter_index;
+var sqlite3_bind_text;
+var sqlite3_changes;
+var sqlite3_clear_bindings;
+var sqlite3_close_v2;
+var sqlite3_column_blob;
+var sqlite3_column_bytes;
+var sqlite3_column_double;
+var sqlite3_column_name;
+var sqlite3_column_text;
+var sqlite3_column_type;
+var sqlite3_create_function_v2;
+var sqlite3_data_count;
+var sqlite3_errmsg;
+var sqlite3_exec;
+var sqlite3_finalize;
+var sqlite3_free;
+var sqlite3_open;
+var sqlite3_prepare_v2;
+var sqlite3_prepare_v2_sqlptr;
+var sqlite3_reset;
+var sqlite3_result_blob;
+var sqlite3_result_double;
+var sqlite3_result_error;
+var sqlite3_result_int;
+var sqlite3_result_int64;
+var sqlite3_result_null;
+var sqlite3_result_text;
+var sqlite3_step;
+var sqlite3_value_blob;
+var sqlite3_value_bytes;
+var sqlite3_value_double;
+var sqlite3_value_int;
+var sqlite3_value_text;
+var sqlite3_value_type;
```

Comment:
Maybe the code would be easier to read if the `var` where situated immediately before the declarations. This would avoid having all these lines.

Reply:
good idea and committed